src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+
+public interface FruitDao {
+
+    void put(String fruitName, int quantity);
+
+    int get(String fruitName);
+
+    Storage getStorage();
```

Comment:
this kinda defeats the purpose and breaks encapsulation if we just return the whole storage, return the entry set instead -     Set<Map.Entry<String, Integer>> getInventoryEntries();